client/jetpack-connect/style.scss

```diff
@@ -656,3 +656,25 @@
 .jetpack-connect__navigation {
 	text-align: center;
 }
+
+.jetpack-connect__plan-info {
+	margin-top: 35px;
+	text-align: center;
+	width: 970px;
+}
+
+.jetpack-connect__plan-info-illustration {
+	margin: 30px 0 10px;
+	max-width: 25%;
```

Comment:
This won't work well for mobile devices. Maybe specify a fixed width in pixels (230px or something)?